server/logger/logger.go

```diff
@@ -0,0 +1,41 @@
+package logger
+
+import (
+	apilogger "github.com/mattermost/mattermost-plugin-api/experimental/bot/logger"
+)
+
+type Logger = apilogger.Logger
+type LogContext = apilogger.LogContext
+
+type LogAPI interface {
```

Comment:
sorry, why is this necessary? Is it because of the `...`? If so, should we make `logger.Logger` compliant with `pluginapi`? I think it's done the way it is for historical reasons (was it p.API compatibility?) cc @larkox

Reply:
It's needed because `logger.Logger` accepts an API containing `LogError`, `LogWarn` etc. and the `pluginapi` exposes `Error` and `Warn` methods. Sure, we can fix that. For now this wrapper is fine to me.